docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,15 +506,27 @@ If you submit a ledger API command (which can have multiple Daml commands), then
 
 Just putting all Daml commands into a single batch command does not help, as you might still create lots of views.
 
+Starting from protocol version 6, Canton creates a view for every action node in the transaction tree if the participants
+that host their informees are not a subset of their parents' informee participants. Therefore, in order to
+minimize the number of views, you should try to minimize either the number of changes of informees or make sure
```

Comment:
Informee changes are actually fine. What you want to avoid is "adding" informees in child nodes.
```suggestion
reduce the number of views, you should try to reduce either the number of times where the set of informees grows or make sure
```